### ISSUE TITLE
Investigate author setting on CMS DocDB

### ISSUE THREAD

**Issue by ericvaandering** [ericvaandering/DocDB#8]
Carrying over authors from one doc to an update is broken for CMS. It's broken at least as far back as 8.8.5, so need to check if this is a CMS only thing or if the problem is really with DocDB code

**Comment by ericvaandering** [comment]
This seems to be a problem with the CMS javascript. Still investigating.

**Comment by lauramengel** [comment]
On FNALSSO version, I do not have to re-enter previously added authors
when I update metadata (including authors) using author search widget.

**Comment by ericvaandering** [comment]
Yeah, I know. Once we get this release out I have to do a real search to figure out what the CMS problem is.
